test_unstructured_ingest/metrics/all-docs-cct.tsv

```diff
@@ -1,4 +1,4 @@
-filename	connector	cct-accuracy	cct-%missing
-handbook-1p.docx	box	0.974	0.03
-example-10k.html	local	0.686	0.04
-IRS-form-1987.pdf	azure	0.783	0.13
+filename	doctype	connector	cct-accuracy	cct-%missing
+handbook-1p.docx	docx	box	0.974	0.03
+example-10k.html	html	local	0.686	0.04
+science-exploration-1p.pptx	pptx	dropbox	0.861	0.09

```

Comment:
If these numbers are all wrong due to my local run, please let me know the best way to update these metrics sheets (ingest text fixtures update?) with the new column as well

Reply:
yes, i'm pretty sure running the ingest test fixtures update (GH workflow) is the proper way to update these

Reply:
Yes, you can run fixture update to correct these. You can also delete those files on local to avoid saving and pushing anything and just update fresh from the fixture update.